internal/scorecard/alpha/tests/bundle_test.go

```diff
@@ -143,3 +143,32 @@ func TestCRDHaveValidation(t *testing.T) {
 		})
 	}
 }
+
+func TestStatusDescriptors(t *testing.T) {
+
+	cases := []struct {
+		bundlePath string
+		state      scapiv1alpha2.State
+		function   func(registry.Bundle) scapiv1alpha2.ScorecardTestResult
+	}{
+		{"../testdata/statusdescriptor/error_bundle", scapiv1alpha2.ErrorState, StatusDescriptorsTest},
```

Comment:
I am not sure if I am able to check the scenarios. 
IHMO: we need to add a name field for each status and then in this name, we add a text as "should pass when xxx" or "should fail when z". E.g https://github.com/camilamacedo86/operator-sdk/blob/master/pkg/ansible/paramconv/paramconv_test.go#L107

Observe that, the test are very helpful for we are able to understand the code and the business rules implemented.